Detectors/MUON/MID/Raw/src/ColumnDataToLocalBoard.cxx

```diff
@@ -0,0 +1,59 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MID/Raw/src/ColumnDataToLocalBoard.cxx
+/// \brief  Converter from ColumnData to raw local boards
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   20 April 2020
+
+#include "MIDRaw/ColumnDataToLocalBoard.h"
+
+#include "MIDBase/DetectorParameters.h"
+#include "MIDRaw/CrateParameters.h"
+
+namespace o2
+{
+namespace mid
+{
+
+void ColumnDataToLocalBoard::process(gsl::span<const ColumnData> data)
+{
+  /// Converts incoming data to FEE format
+  mLocalBoardsMap.clear();
+  mGBTMap.clear();
+
+  // First fill the map with the active local boards.
+  // Each local board gets a unique id.
+  for (auto& col : data) {
+    for (int iline = 0; iline < 4; ++iline) {
+      if (col.getBendPattern(iline) == 0) {
+        continue;
+      }
+      auto uniqueLocId = mCrateMapper.deLocalBoardToRO(col.deId, col.columnId, iline);
+      auto& roData = mLocalBoardsMap[uniqueLocId];
+      roData.statusWord = raw::sSTARTBIT | raw::sCARDTYPE;
+      roData.boardId = crateparams::getLocId(uniqueLocId);
+      int ich = detparams::getChamber(col.deId);
+      roData.firedChambers |= (1 << ich);
+      roData.patternsBP[ich] = col.getBendPattern(iline);
+      roData.patternsNBP[ich] = col.getNonBendPattern();
+    }
+  }
+
+  // Then ranges the boards per GBT link
```

Comment:
ranges the boards ??

Reply:
It was a direct translation from the French "ranger"...but it has a different meaning in English :D . Ok I changed it into the clearer:
// Then group the boards belonging to the same GBT link